src/main/java/spoon/support/compiler/jdt/JDTTreeBuilderHelper.java

```diff
@@ -826,40 +827,18 @@ CtModuleRequirement createModuleRequirement(RequiresStatement requiresStatement)
 		return moduleRequirement;
 	}
 
-	CtPackageExport createModuleExport(ExportsStatement exportsStatement) {
-		String packageName = new String(exportsStatement.pkgName);
-		int sourceStart = exportsStatement.sourceStart;
-		int sourceEnd = exportsStatement.sourceEnd;
+	CtPackageExport createModuleExport(PackageVisibilityStatement statement) {
```

Comment:
Am I correct in my assumption a PackageVisibilityStatement is a supertype of ExportsStatement and OpensStatement?